packages/marketplace/src/services/utils.ts

```diff
@@ -7,6 +7,11 @@ export const generateHeader = (marketplaceApiKey): StringMap => ({
   'X-Api-Key': marketplaceApiKey,
 })
 
+export const generateHeaderV2 = (marketplaceApiKey): StringMap => ({
```

Comment:
I think should change to `generateHeaderWithApiV2` to avoid confusing

Reply:
I changed it to generateHeaderWithApiV2